Frameworks/include/CollectionHelpers.h

```diff
@@ -80,5 +81,17 @@ HRESULT WRLToNSCollection(const Microsoft::WRL::ComPtr<T>& collection,
                           typename Private::CollectionType<T>::NSEquivalentType* __autoreleasing* pNSCollection) {
     return WRLToNSCollection(collection.Get(), pNSCollection);
 }
+
+static HRESULT ToastUserInputToNSDictionary(WFCValueSet* userInput, NSMutableDictionary** dict) {
```

Comment:
My concern here is that we're bringing in a dependency on Projections for an otherwise strictly-WRL header.

Is it possible to coerce this into another type of WRLToNSCollection, with another specialization?